### ISSUE TITLE
Why html-page hasn't favicon if I use webpack-dev-server?

### ISSUE THREAD

**Issue by Andrey-Bushman** [jantimon/favicons-webpack-plugin#246]
If I run `npm run build:dev` or `npm run build:prod` then I see favicon when I open the result html-file. But if I use `webpack-dev-server` through the command `npm run start:dev` then opened html page hasn't favicon. Why?

Settings in my **package.json**:

```
 "scripts": {
    "build:dev": "npx webpack --env mode=development",
    "build:prod": "npx webpack --env mode=production",
    "start:dev": "npx webpack serve --env mode=development --open"
  },
...
  "devDependencies": {
    "favicons-webpack-plugin": "^4.2.0",
    "html-webpack-plugin": "^4.5.1",
    "webpack": "^5.17.0",
    "webpack-cli": "^4.4.0",
    "webpack-dev-server": "^3.11.2",
    ...
  }
```

My code in **webpack.config.js** (in the `plugins` section):

```
new FaviconsWebpackPlugin({
    logo: 'assets/images/favicon.ico',
    prefix: '/assets/images',
    publicPath: '.'
}),
```

**Comment by jantimon** [comment]
https://github.com/jantimon/favicons-webpack-plugin#compilation-modes

**Comment by jahed** [comment]
This seems to be a bug. When using `prefix`, in development, it needs to end in a `/`. Without it the prefix will combine with the icon name (e.g. `prefixicon.png`) and lead to a 404. However, in production, adding a `/` causes a double `//` (`prefix//icon.png`).

**Comment by jantimon** [comment]
This should be fixed with 5.0 - is it still an issue?
